console/queue_publisher.py

```diff
@@ -0,0 +1,59 @@
+import pika
+
```

Comment:
We could use the sdc-rabbit library instead of having this class.

https://github.com/ONSdigital/sdc-rabbit

For example, have a look at the notification publisher here:
https://github.com/ONSdigital/sdx-collect/blob/master/app/response_processor.py